src/Raven.Studio/typescript/components/pages/resources/databases/partials/create/formBackup/createDatabaseFromBackupDataUtils.ts

```diff
@@ -177,13 +177,17 @@ function getSourceDto(
     encryptionDataIsEncrypted: boolean,
     encryptionDataKey: string
 ): SelectedSourceDto & Pick<CreateDatabaseFromBackupDto, "BackupLocation" | "Settings"> {
+    const backupLocation = isSharded
```

Comment:
For sharding, we don't need this?

Reply:
For shards the value is in
`ShardRestoreSettings.Shards[].FolderName`

Reply:
Because this value is different for each restore point